Patch: 01_defaults.debian.dpatch
Author: Tollef Fog Heen <tfheen@debian.org>
Set sane defaults in Defaults.py.in
Index: mailman-2.1.12/Mailman/Defaults.py.in
===================================================================
--- mailman-2.1.12.orig/Mailman/Defaults.py.in	2009-03-02 20:41:57.000000000 +0100
+++ mailman-2.1.12/Mailman/Defaults.py.in	2009-03-02 20:49:55.000000000 +0100
@@ -54,7 +54,7 @@
 # disable Mailman's logo footer altogther, hack
 # Mailman/htmlformat.py:MailmanLogo(), which also contains the hardcoded links
 # and image names.
-IMAGE_LOGOS = '/icons/'
+IMAGE_LOGOS = '/doc/mailman/images/'
 
 # The name of the Mailman favicon
 SHORTCUT_ICON = 'mm-icon.png'
@@ -85,7 +85,7 @@
 # because otherwise the default mappings won't be correct.
 DEFAULT_EMAIL_HOST = '@MAILHOST@'
 DEFAULT_URL_HOST = '@URLHOST@'
-DEFAULT_URL_PATTERN = 'http://%s/mailman/'
+DEFAULT_URL_PATTERN = 'http://%s/cgi-bin/mailman/'
 
 # DEFAULT_HOST_NAME has been replaced with DEFAULT_EMAIL_HOST, however some
 # sites may have the former in their mm_cfg.py files.  If so, we'll believe
@@ -1352,7 +1352,7 @@
 MAILDIR_DIR     = os.path.join(QUEUE_DIR, 'maildir')
 
 # Other useful files
-PIDFILE = os.path.join(DATA_DIR, 'master-qrunner.pid')
+PIDFILE = "/var/run/mailman/mailman.pid"
 SITE_PW_FILE = os.path.join(DATA_DIR, 'adm.pw')
 LISTCREATOR_PW_FILE = os.path.join(DATA_DIR, 'creator.pw')
 
